core/base/src/TApplication.cxx

```diff
@@ -1732,6 +1733,12 @@ Longptr_t TApplication::ProcessLine(const char *line, Bool_t sync, Int_t *err)
 #endif
    }
 
+   if (!strcmp(line, ".libraries")) {
```

Comment:
This resolves the issue.  It would probably be better yet, if this was part of `Cling` proper.

Reply:
@vgvassilev is this something which can be interesting in general?

Reply:
Probably not. It’s better here for now.